wallet-new/src/Cardano/Wallet/Kernel/DB/Spec/Update.hs

```diff
@@ -83,6 +84,10 @@ newPending tx = do
             let unavailableInputs = txAuxInputSet tx' `Set.difference` utxoInputs available_
             throwError $ NewPendingInputsUnavailable (Set.map InDb unavailableInputs)
 
+evictPending :: InDb (Set Core.TxId) -> Checkpoints -> Checkpoints
+evictPending txids checkpoints =
+    checkpoints & (over currentPending (removePending (txids ^. fromDb)))
```

Comment:
RTFM. https://cardanodocs.com/technical/formal-specification-for-a-cardano-wallet/.

imp

:troll:

Also, let's stick with the naming in the spec.